NewEvaluation.css

```diff
@@ -68,3 +92,107 @@ body{
   	width:100%;  background-color: #dfdfdf; text-align: center; vertical-align: center; padding-bottom: 1em;
   	color:#828384;
 }
+
+
+.legend-box legend{
```

Comment:
Hmm... so there's a `legend` inside a `.legend-box`. Fair. But could we make it so the class name is more generic and less tightly coupled to the markup?